src/test/java/io/pravega/connectors/flink/source/FlinkPravegaSourceITCase.java

```diff
@@ -131,8 +131,7 @@ private static void runTest(
             // the Pravega reader
             final PravegaSource<Integer> pravegaSource = PravegaSource.<Integer>builder()
                     .forStream(streamName)
-                    .enableMetrics(false)
-                    .withPravegaConfig(SETUP_UTILS.getPravegaConfig())
+                    .withPravegaClientConfig(SETUP_UTILS.getPravegaClientConfig())
```

Comment:
In FLIP-27 tests, we should set the config one by one instead of using a `Configuration` here, and even I think it's fine for now to remove the `withPravegaClientConfig()`. We can add it back when we need this to integrate the table API in the future.